### ISSUE TITLE
Error processing archive oracle-xe_11.2.0-2_amd64.deb

### ISSUE THREAD

**Issue by schunt1** [OraOpenSource/OXAR#191]
Any clues why I'm now getting this error?
![xe fail](https://cloud.githubusercontent.com/assets/2545233/19019439/7f1052d2-8880-11e6-9e7d-0f6124664cac.PNG)

These are the zip files
![sw](https://cloud.githubusercontent.com/assets/2545233/19019450/bfe6fa36-8880-11e6-81ed-b15bb1c0bac1.PNG)

**Comment by tschf** [comment]
One of the programs installed in the initial stages on Debian systems is
`alien` - this program is used to convert from an rpm to a deb package
file. See:
https://github.com/OraOpenSource/OXAR/blob/master/scripts/packages.sh#L37.
The first error in your output suggests it hasn't been installed though.
The build script seems to be set to use the following as a distro test for
Debian: `elif [ -n "$(command -v apt-get)" ]; then` to test if on Debian
(since apt-get is the utility to handle packages on Debian systems).

I wonder if `command` is somehow not available in your system. What happens
when you run `command -v apt-get`?

Not sure if it helps, there's also a flag to show all output to the
terminal: -v. The default behaviour is to suppress standard output.

**Comment by schunt1** [comment]
![capture](https://cloud.githubusercontent.com/assets/2545233/19019989/8b31702e-8893-11e6-89d6-65a97b55912d.PNG)

**Comment by schunt1** [comment]
How about I reset my environment, go though the install until the actual build step and then email you the connection details so you can watch go through the install process?  It's a standard Linux dist on Azure, so would be good to get it working.

**Comment by martindsouza** [comment]
@schunt1 yes I recommend starting from scratch. OXAR was always made for vanilla systems. The mail goal is to take cloud machines and build up a machine from scratch quickly.

Let us know how it goes. I'll leave this ticket open for the time being.

**Comment by tschf** [comment]
@martindsouza Looks like this is a problem with the SSL additions. Specifically with the certbot dependency:

```
trent@TRENTXE:~$ sudo apt-get install certbot
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 certbot : Depends: python-certbot (= 0.8.1-2~bpo8+1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Thus none of the dependencies are installed

**Comment by tschf** [comment]
Some discussions about the issue:

* https://community.letsencrypt.org/t/problem-while-installing-certbot/16314
* https://community.letsencrypt.org/t/unable-to-install-certbot/17916
* https://community.letsencrypt.org/t/can-not-install-on-debian-jessie-backports/17935

As from one of the resources, I worked around this issue on Debian by enabling backports and installed from backports before running the oxar build script. You may have a better more suitable approach, Martin. But @schunt1, here's a workaround in the meantime - before building:

```bash
sudo -s
echo "deb http://ftp.debian.org/debian jessie-backports main" >> /etc/apt/sources.list
apt-get update
apt-get -t jessie-backports install "python-certbot"
```

as per: https://backports.debian.org/Instructions/

**Comment by martindsouza** [comment]
@tschf I tested with CentOS when I added certbot. I don't have any Debian OSs on my system. Do you want to apply your suggested fix to the system then for Debian users?

**Comment by martindsouza** [comment]
We're going to temporarily drop Debian support (see #198). Please use CentOS
